apps/google-docs/src/locations/Page.tsx

```diff
@@ -138,8 +139,8 @@ const Page = () => {
 
   return (
     <>
-      <UploadDocumentModal
-        sdk={sdk}
```

Comment:
We could delete the `UploadDocumentModal` and `GoogleDocUploader` components now that we have this new `SelectDocumentModal` component and the other two aren't used in the app.